internal/cluster/distribution/eks/eksprovider/workflow/templates.go

```diff
@@ -60,7 +62,18 @@ func getEksCloudFormationTemplate(name string) (string, error) {
 		return "", errors.WrapIf(err, fmt.Sprintf("failed to read CloudFormation template content from %s", templatePath))
 	}
 
-	return string(content), nil
+	t, err := template.New("cf").Parse(string(content))
+	if err != nil {
+		return "", errors.WrapIf(err, fmt.Sprintf("failed to parse CloudFormation template content from %s", templatePath))
+	}
+
+	buffer := bytes.NewBuffer(nil)
+	err = t.Execute(buffer, global.Config)
```

Comment:
Ideally the enterprise flag should not appear directly in templates and code. The template data could be something like:

```
data := map[string]interface{}{
    "UpdatePolicyEnabled": !config.Pipeline.Enterprise,
}
```